### ISSUE TITLE
The Acc@0.5 of Visual Grounding does not match.

### ISSUE THREAD

**Issue by zytx121** [lx709/VRSBench#4]
I used 8-card V100, batch=32 (4 * 8), and trained LLaVA-v15-7B with hyperparameters according to the paper, but the accuracy was only 14.26 and did not match the 36.3 in the paper. @lx709 May I ask if you can help me check where my training configuration went wrong? What does the term 'batch size is standardized at 32' mean in the paper? Is it per card or total?

```
#!/bin/bash
srun -p mediasuper  --gres=gpu:8 --nodes=1 --ntasks-per-node=1  --kill-on-bad-exit=1 \
deepspeed llava/train/train_xformers.py \
    --lora_enable True --lora_r 64 --lora_alpha 16 --mm_projector_lr 2e-5 \
    --deepspeed ./scripts/zero2.json \
    --model_name_or_path data/llava-v1.5-7b/ \
    --version v1 \
    --data_path data/vrsbench/VRSBench_train.json \
    --image_folder data/vrsbench/Images_train \
    --vision_tower openai/clip-vit-large-patch14-336 \
    --mm_projector_type mlp2x_gelu \
    --pretrain_mm_mlp_adapter data/llava-v1.5-mlp2x-336px-pretrain-vicuna-7b-v1.5/mm_projector.bin \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end False \
    --mm_use_im_patch_token False \
    --image_aspect_ratio pad \
    --group_by_modality_length True \
    --bf16 False \
    --fp16 True \
    --output_dir ./checkpoints/llava-v1.5-7b-vrsbench \
    --num_train_epochs 5 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 50000 \
    --save_total_limit 1 \
    --learning_rate 2e-4 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 False \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --dataloader_num_workers 16 \
    --lazy_preprocess True \
    --report_to wandb
```

**Comment by zytx121** [comment]
The accuracy is correct, thank you for the open source!
